src/services/matugen/index.ts

```diff
@@ -33,6 +33,13 @@ export async function generateMatugenColors(): Promise<MatugenColors | undefined
         const contents = await bash(
             `matugen image --dry-run -q ${wallpaperPath} -t scheme-${scheme_type.get()} --contrast ${normalizedContrast} --json hex`,
         );
+        const themeStatus= await bash(
```

Comment:
```suggestion
        await bash(
```
We can remove the constant since it's not used anywhere.